tools/rapids-telemetry-record

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Check for at least 2 arguments: filename and command
+if [ "$#" -lt 2 ]; then
+    echo "Usage: $0 filename command [arguments...]"
+    exit 1
+fi
+
+output_file="$1"
+shift
+
+# TODO: Make this a temporary path rather than writing in the GITHUB_WORKSPACE.
```

Comment:
@msarahan I'd like your thoughts on this. Could we move telemetry artifacts to use a temporary directory? Thankfully, merging this PR in the current state should isolate those changes from the library repository level -- it would be only a matter of aligning `shared-actions` and `gha-tools`.

Reply:
Perhaps we could even make this run `mkdir -p` in a way that is idempotent, and removes the need for `rapids-telemetry-setup`. And/or we could write the telemetry artifact path to `${GITHUB_ENV}` for use in later steps? Not sure.

Reply:
Yes, you can write the output path to GITHUB_ENV. That should work without issue, as these paths are not carried between workflows.

Reply:
I’ll see if we can revisit this in a follow-up. This is sufficient for now.